gambling.js

```diff
@@ -45,6 +80,44 @@ exports.dischargeScrubBubble = function (userID, botSpam) {
     const title = dropped + ' Scrubbing ' + msg + ' arrived for duty!';
     util.sendEmbedMessage(title, null, c.BUBBLE_IMAGES[droppedImg-1]);
 }
+/**
+ * @param {Number} userID - user's ID dispatching Scrubs
+ * @param {Number} amount - amount the user wants to dispatch
+ * @param {Number} dispatchee - user ID of the receiver for Dispatch Command
+ */
+function dispatchScrubs(userID, amount, dispatchee){
+    if(ledger[userID] && ledger[dispatchee] && ledger[userID].armySize >= amount){
+        addToArmy(userID, amount, 'take');
+        addToArmy(dispatchee, amount, 'add');
+
+        description = '<@!' + dispatchee + '>'+ ' '+ amount +  ' Scrubbing Bubble' + maybeGetPlural(amount) + ' have been dispatched to you by' + '<@!' + userID + '>';
+        util.sendEmbedMessage(null, description);
+    }
+    
+}
+/**
```

Comment:
add description of function above @params as seen above in various other function's JSDoc